src/view/treeNodes/pullRequestNode.ts

```diff
@@ -281,7 +281,7 @@ export class PRNode extends TreeNode implements vscode.CommentingRangeProvider2
 		const currentBranchIsForThisPR = this.pullRequestModel.equals(this._folderReposManager.activePullRequest);
 
 		const { title, number, author, isDraft, html_url } = this.pullRequestModel;
-
+		const labelTitle = this.pullRequestModel.title.length > 50 ? `${this.pullRequestModel.title.substring(0, 50)}...` : this.pullRequestModel.title;
```

Comment:
Nit: could the constant 50, which is currently in two separate files, be pulled out to a common file?